src/shared/components/barGraph/BarGraph.tsx

```diff
@@ -129,74 +132,137 @@ export default class BarGraph extends React.Component<IBarGraphProps, {colors: s
+                    // Set caret Position
+                    tooltipEl.classList.remove('above', 'below', 'no-transform');
+                    if (tooltipModel.yAlign) {
+                        tooltipEl.classList.add(tooltipModel.yAlign);
+                        // tooltipEl.classList.add('below');
+                    } else {
+                        tooltipEl.classList.add('no-transform');
+                    }
 
-            xAxis: {categories: cancerTypeStudiesArray.slice(0, 20).map((cancer) => (cancer.shortName)),
-                    labels: {style: {fontSize: '10px'}}},
+                    function getBody(bodyItem:BodyItem) {
+                        return bodyItem.lines;
+                    }
 
-            yAxis: {
-                allowDecimals: false,
-                min: 0,
-                // max: 6000,
-                title: {text: ''}
-            },
+                    // Set Text
+                    if (tooltipModel.body) {
+                        const titleLines = tooltipModel.title || [];
+                        const bodyLines = tooltipModel.body.map(getBody);
 
-            legend: {enabled: false},
-
-            tooltip: {
-                useHTML: true,
-                backgroundColor: '#ffffff',
-                formatter: function() {
-                    return `
-                            <div style="font-size:11px;">
-                                ${this.x}: ${this.point.stackTotal} cases
-                                <br/> 
-                                <span style="width: 200px !important; font-size:12px; color:${this.point.color}; overflow:auto; white-space:normal !important;" >
-                                    ${this.series.name}: <b>${this.point.y} cases</b>
-                                </span>
-                            </div>
-                          `;
-                }
-            },
+                        let innerHtml = '<thead>';
+
+                        titleLines.forEach(function(title: string) {
+                            innerHtml += '<tr><th>' + title + ': ' + datasets[tooltipModel.dataPoints[0].datasetIndex].total + ' cases</th></tr>';
+                        });
+                        innerHtml += '</thead><tbody>';
+
+                        bodyLines.forEach(function(body:string, i:number) {
+                            tooltipEl!.style.borderColor = datasets[tooltipModel.dataPoints[0].datasetIndex].backgroundColor;
 
-            plotOptions: {
-                series: {
-                    stacking: 'normal',
-                    cursor: 'pointer',
-                    events: {
-                        click: function() {
-                            window.location.href= 'study?id=' + this.options.studyId, + '#summary';
-                        }
+                            const colors = tooltipModel.labelColors[i];
+                            let style = 'background-color: ' + tooltipModel.labelColors[0].backgroundColor;
```

Comment:
use template literals for this.  or perhaps even a component.  avoid concatenation except in extremely simple cases.